tests/Remove-DbaDbRole.Tests.ps1

```diff
@@ -0,0 +1,87 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandpath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tags "UnitTests" {
+    Context "Validate parameters" {
+        [object[]]$params = (Get-Command $CommandName).Parameters.Keys | Where-Object {$_ -notin ('whatif', 'confirm')}
+        [object[]]$knownParameters = 'SqlInstance', 'SqlCredential', 'Database', 'ExcludeDatabase', 'Role', 'ExcludeRole', 'IncludeSystemDbs', 'InputObject', 'EnableException'
+        $knownParameters += [System.Management.Automation.PSCmdlet]::CommonParameters
+        It "Should only contain our specific parameters" {
+            (@(Compare-Object -ReferenceObject ($knownParameters | Where-Object {$_}) -DifferenceObject $params).Count ) | Should Be 0
+        }
+    }
+}
+
+Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
+    BeforeAll {
+        $server = Connect-DbaInstance -SqlInstance $script:instance2
+        $role1 = "dbatoolssci_role1_$(Get-Random)"
+        $role2 = "dbatoolssci_role2_$(Get-Random)"
+        $dbname1 = "dbatoolsci_$(Get-Random)"
+
+        $null = New-DbaDatabase -SqlInstance $script:instance2 -Name $dbname1 -Owner sa
+        #$null = New-DbaDatabase -SqlInstance $script:instance2 -Name $dbname2 -Owner sa
+
+
+        #$null = $server.Query("CREATE ROLE $role1", 'msdb')
+    }
+    AfterAll {
+        $null = Remove-DbaDatabase -SqlInstance $script:instance1 -Database $dbname1 -confirm:$false
```

Comment:
Should be referencing instance2

Also, commented out code should be removed once testing is completed.